tensorflow/python/keras/utils/tf_utils_test.py

```diff
@@ -20,14 +20,82 @@
 
 from tensorflow.python import keras
 from tensorflow.python.eager import context
+from tensorflow.python.framework import constant_op
 from tensorflow.python.framework import ops
 from tensorflow.python.framework import sparse_tensor
 from tensorflow.python.framework import test_util
 from tensorflow.python.keras.utils import tf_utils
+from tensorflow.python.ops import array_ops
 from tensorflow.python.ops import variables
 from tensorflow.python.platform import test
 
 
+class ConstantValueTest(test.TestCase):
+
```

Comment:
I think we need to test Variable case as that is the only additional use case tf_utils.smart_cond is solving. Otherwise we just redirect to the underlying smart_cond API.